src/server/script_mgr.cc

```diff
@@ -5,33 +5,61 @@
 #include "server/script_mgr.h"
 
 #include <absl/cleanup/cleanup.h>
+#include <absl/strings/ascii.h>
+#include <absl/strings/match.h>
+#include <absl/strings/numbers.h>
 #include <absl/strings/str_cat.h>
+#include <absl/strings/str_split.h>
 
+#include <string>
+
+#include "base/flags.h"
 #include "base/logging.h"
 #include "core/interpreter.h"
 #include "facade/error.h"
 #include "server/engine_shard_set.h"
 #include "server/server_state.h"
 
+ABSL_FLAG(std::string, default_script_config, "",
+          "Default config for scripts: non-atomic to disable atomicity, undeclared-keys to allow "
```

Comment:
maybe call it `default_lua_config`?
also, in the description show actually show how to set the script since you allow here multiple value.
Moreover, I would use `allow-undeclared-keys` and `disable-atomic` as values to make it even more clear.

Reply:
Even more I would repeat the explanation in the commit description with the example of how you run dragonfly in various modes.

Reply:
its `disable-atomicity` then